app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,13 @@
+class CommentsController < ApplicationController
+def index
+end
+def new
+@comment=Comment.new
+end
+def create
+  @comment = Comment.new(params[:comment])
+ 
+  @comment.save
+  redirect_to @comment
```

Comment:
redirect_to string parameter leta h, jahan redirect krna h
aur wo string rails helper se bna
example root_path ek rails helper h. (helper matlab helping function)
`puts root_path` ========>  returns "/"
`redirect_to root_path`
will redirect to "/" matlab localhost:3000/
ise aise bhi likh sakte h
`redirect_to "/"`
same cheez h
ab tujhe jahan redirect krna h wahan ka path (relative path, localhost nahi) de.